communication guidelines.md

```diff
@@ -0,0 +1,77 @@
+## Purpose:
+[GitHub Actions](https://docs.github.com/en/actions/learn-github-actions) gives you the flexibility to build an automated software development lifecycle workflow. With [GitHub Actions for Azure](https://docs.microsoft.com/en-gb/azure/developer/github/) you can create workflows that you can set up in your repository to build, test, package, release and deploy to Azure
+This document is to set communication guidelines, processes for making GitHub Actions ready for the external community. We want to set a robust framework for communication, and governance for anything related to Azure Actions, for enabling both internal and external users to be always updated on any change in this area.
```

Comment:
The purpose of the document should have only communication guidelines and not the processes

Reply:
Updated